software/webdb.yml

```
@@ -0,0 +1,39 @@
+---
+name: Addition
+about: Add new software to the list.
+title: Add WebDB
+labels: addition
+assignees: ''
+
+---
+
+Thanks for taking the time to suggest an addition to awesome-selfhosted! 
+
+Please fill out information below (all fields are mandatory unless noted otherwise):
+
```

```suggestion
---
- name: Addition
- about: Add new software to the list.
- title: Add WebDB
- labels: addition
- assignees: ''

---

- Thanks for taking the time to suggest an addition to awesome-selfhosted! 

- Please fill out information below (all fields are mandatory unless noted otherwise):

```
this is not needed either.